### PR TITLE
Athena/branch3

### PR DESCRIPTION
**Pull request by ath-chen**
todo: add ajax call to when we hit "x" button 

- added ajax call for when you hit "finish" button
- fixed widths in merchant directory --> hero spans full width now & content is aligned 

Screen Shot 2020-05-19 at 12 43 10 AM

Screen Shot 2020-05-19 at 12 43 23 AM

### CODE REVIEW

**Comment by jtmckibb** [comment]
wait whoa this is amazing! is it possible to do the same thing for whenever someone clicks the "translation button" in the header?

File: src/components/MerchantsPage/styles.module.scss [hunk 1]
```diff
@@ -57,9 +44,16 @@
...
   height: 55%;
-  width: 100%;
   object-fit: cover;
 
+  // to break the image out of the parent container
+  width: 100vw;
+  position: relative;
+  left: 50%;
+  right: 50%;
+  margin-left: -50vw;
```
[the remark this answers is not in the document]
Comment (jtmckibb) [reply src/components/MerchantsPage/styles.module.scss:54]: seems like we need to re-arrange the view hierarchy

File: src/components/Modal/Modal.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export const Modal = (props: Props) => {
 
   const closeModal = (e: any) => {
     e.preventDefault();
-    dispatch({ type: CLOSE_MODAL, payload: undefined });
+    dispatch({ type: CLOSE_MODAL, payload: -1 });
```
Comment (jtmckibb) [inline src/components/Modal/Modal.tsx:33]: why this change?

File: src/components/ModalConfirmation/ModalConfirmation.tsx [hunk 3]
```diff
@@ -21,10 +25,17 @@ const ModalConfirmation = (props: Props) => {
       case 'buy_meal':
         return `We appreciate your support for ${props.sellerName} and for those in need! Please check your email for your receipt.`;
       default:
```
Comment (jtmckibb) [inline src/components/ModalConfirmation/ModalConfirmation.tsx:27]: do we need a default case?

File: src/components/ModalPayment/SquareModal.tsx [hunk 4]
```diff
@@ -141,21 +141,30 @@ const SquareModal = ({
...
-              making a donation to ${sellerName}. No goods or services were
-              exchanged for this donation.`,
-    donationPool: `By proceeding with your transaction, you understand that 
-                  you are making a donation to all merchants partnered with Send Chinatown Love 
-                  Inc. The full donation pool will be split among these merchants. No goods or 
-                  services were exchanged for this donation`,
+  const disclaimerLanguage = (type: any) => {
+    if (sellerId === 'send-chinatown-love') type = 'donation-pool';
+
+    switch (type) {
```
Comment (jtmckibb) [inline src/components/ModalPayment/SquareModal.tsx:147]: is it possible to translate these?

File: src/components/ModalConfirmation/ModalConfirmation.tsx (hunk 3 above)
Comment (jacoblb64) [reply src/components/ModalConfirmation/ModalConfirmation.tsx:27]: if this is an unexpected case, should we log here?

File: src/components/ModalPayment/SquareModal.tsx [hunk 5]
```diff
@@ -141,21 +141,30 @@ const SquareModal = ({
...
-              represent the final balance. Final balance information is subject to 
-              ${sellerName}'s most recent records.`,
-    donation: `By proceeding with your transaction, you understand that you are
-              making a donation to ${sellerName}. No goods or services were
-              exchanged for this donation.`,
-    donationPool: `By proceeding with your transaction, you understand that 
-                  you are making a donation to all merchants partnered with Send Chinatown Love 
-                  Inc. The full donation pool will be split among these merchants. No goods or 
-                  services were exchanged for this donation`,
+  const disclaimerLanguage = (type: any) => {
```
Comment (jacoblb64) [inline src/components/ModalPayment/SquareModal.tsx:144]: nit: if we're typing this, shouldn't it be `string`?
